fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -1073,9 +1075,11 @@ abstract class BaseBrowserFragment :
 
     @VisibleForTesting
     internal fun shouldPullToRefreshBeEnabled(inFullScreen: Boolean): Boolean {
-        return FeatureFlags.pullToRefreshEnabled &&
+        val shouldPullToRefreshBeEnabled = FeatureFlags.pullToRefreshEnabled &&
```

Comment:
I think we want to add telemetry for PTR _changes_ to understand if people turning it off (if it's already on by default), so we want to add a probe at the UI settings level so that we know when a user turns it on/off.

@travis79 Do we need do another data-review if we need to change the name and description of the event? I think will have to do that when we move this to the right place.

Reply:
If the event is still answering the same question, this seems like a semantic change to me. Unless you feel like it would substantially change the original request, then the original data-review can stand.

Reply:
Please do update the request with the appropriate names and I'll do the same in my review comment.

Reply:
Done